nav2_mppi_controller/include/nav2_mppi_controller/critics/path_align_critic.hpp

```diff
@@ -52,6 +54,9 @@ class PathAlignCritic : public CriticFunction
   bool use_path_orientations_{false};
   unsigned int power_{0};
   float weight_{0};
+
+  bool visualize_{false};
+  nav2::Publisher<geometry_msgs::msg::PoseStamped>::SharedPtr target_pose_pub_;
```

Comment:
Maybe this is better as a Marker sphere.

Currently:

<img width="326" height="421" alt="Image" src="https://github.com/user-attachments/assets/6460b18c-e31a-4bd1-be70-d10e34addd8c" />